### PR TITLE
Interval

### PR DESCRIPTION
**Pull request by lisa-von-mond**
note:  layout does not work yet on mobile devices, please view on desktop.

Changed some details in the concept of cockpit functionality. Rules can be read direkt in the deployment, playing first label should be possible now :)

thx in advance for review!

### CODE REVIEW

File: components/cockpit.jsx [hunk 1]
```diff
@@ -1,98 +1,89 @@
 import styled, {css} from "styled-components";
```
Comment (Carolin-Weitkamp) [inline components/cockpit.jsx:1]: He, i think you don´t have to add the css in the curly Brackets.
Like this: import styled from 'styled-components'; :
it shoukd be fine

File: components/cockpit.jsx [hunk 2]
```diff
@@ -111,6 +102,7 @@ width:auto;
 padding:20px;
 background-image: linear-gradient(to right, #f78ca0 0%, #f9748f 19%, #fd868c 60%, #fe9a8b 100%);
```
Comment (Carolin-Weitkamp) [inline components/cockpit.jsx:103]: i think its not neccessary to use background image.
Just background should be enough. ?

File: components/legend.jsx [hunk 3]
```diff
@@ -4,32 +4,36 @@ export function Legend(){
...
+        <p><i>2 UP</i> will be resolved as <i>UP UP</i>. 2 CLOSE LEFT will be resolved as <i>CLOSE LEFT CLOSE LEFT. 3 UP LEFT LEFT</i> will be resolved as <i>UP LEFT LEFT UP LEFT LEFT UP LEFT LEFT</i>. Deleting last command is only possible in green line.
+        </p>
+        <p>*****</p>
         <p>COUNTER</p>
-        <p>This is a dummy, in the real game you will only have 14 moves to complete your mission. So if counter is warning, you normally would have failed =)</p>
-        <p>***</p>
+        <p>You have 12 moves to complete your mission (actually game goes on after 12 moves but counter will warn)</p>
+        <p>Every command in the green command line (also a bundled command) will be counted as one. Impossible commands will be counted normally but ignored in movements.</p>
+        <p>*****</p>
         <p>Have a nice journey!</p>
```
Comment (Carolin-Weitkamp) [inline components/legend.jsx:27]: super nice!

File: components/cockpit.jsx (hunk 2 above)
Comment (lisa-von-mond) [reply components/cockpit.jsx:103]: changed it!

**Pull request merged**